feature/interface/singleton/tests/singleton_with_breakouts/singleton_with_breakouts_test.go

```diff
@@ -0,0 +1,212 @@
+package singleton_with_breakouts
+
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import (
+	"context"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	gnps "github.com/openconfig/gnoi/system"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/testt"
+	"github.com/openconfig/ygot/ygot"
+)
+
+const (
+	maxRebootTime   = 900
+	maxCompWaitTime = 600
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+// Method to run baseLine test
+func baseLineTest(t *testing.T, dut *ondatra.DUTDevice, portsConfigured map[string]bool, rebootCheck bool) {
+	if !rebootCheck {
+		configureDUT(t, dut, portsConfigured)
+	}
+	verifyConsistentPMD(t, dut, portsConfigured)
+}
+
+// Method to configure DUT interfaces along with breakout configurations
+func configureDUT(t *testing.T, dut *ondatra.DUTDevice, portsConfigured map[string]bool) {
+	topo := gnmi.OC()
+	for _, port := range dut.Ports() {
+		portsConfigured[port.Name()] = false
+		if port.PMD() == ondatra.PMD400GBASEDR4 {
+			numOfBreakouts := uint8(4)
+			hardwarePort := gnmi.Get(t, dut, gnmi.OC().Interface(port.Name()).HardwarePort().State())
+			bmode := &oc.Component_Port_BreakoutMode{}
+			bmp := topo.Component(hardwarePort).Port().BreakoutMode()
+			group := bmode.GetOrCreateGroup(0)
```

Comment:
could you please change the group id to 1 instead of 0 , as i see in other scritps default group expected is index 1 for example  https://github.com/openconfig/featureprofiles/blob/main/feature/system/gnmi/metadata/tests/large_set_consistency_test/large_set_consistency_test.go#L79

Reply:
Addressed in the latest commit

Reply:
Resolved - Changed the breakout mode group id to 1